src/Client/BuzzHouse/Generator/FuzzConfig.cpp

```diff
@@ -0,0 +1,232 @@
+        {"time_to_run", [&](const JSONObjectType & value) { time_to_run = static_cast<uint32_t>(value.getUInt64()); }},
+        {"fuzz_floating_points", [&](const JSONObjectType & value) { fuzz_floating_points = value.getBool(); }},
+        {"test_with_fill", [&](const JSONObjectType & value) { test_with_fill = value.getBool(); }},
+        {"use_dump_table_oracle", [&](const JSONObjectType & value) { use_dump_table_oracle = value.getBool(); }},
+        {"clickhouse", [&](const JSONObjectType & value) { clickhouse_server = loadServerCredentials(value, "clickhouse", 9004, 9005); }},
+        {"mysql", [&](const JSONObjectType & value) { mysql_server = loadServerCredentials(value, "mysql", 3306, 3306); }},
+        {"postgresql", [&](const JSONObjectType & value) { postgresql_server = loadServerCredentials(value, "postgresql", 5432); }},
+        {"sqlite", [&](const JSONObjectType & value) { sqlite_server = loadServerCredentials(value, "sqlite", 0); }},
+        {"mongodb", [&](const JSONObjectType & value) { mongodb_server = loadServerCredentials(value, "mongodb", 27017); }},
+        {"redis", [&](const JSONObjectType & value) { redis_server = loadServerCredentials(value, "redis", 6379); }},
+        {"minio", [&](const JSONObjectType & value) { minio_server = loadServerCredentials(value, "minio", 9000); }},
+        {"disabled_types",
+         [&](const JSONObjectType & value)
+         {
+             std::string input = std::string(value.getString());
+             std::transform(input.begin(), input.end(), input.begin(), ::tolower);
+             const auto & split = splitString(input, ',');
+
+             std::map<std::string, uint32_t> type_entries
+                 = {{"bool", allow_bool},
+                    {"uint", allow_unsigned_int},
+                    {"int8", allow_int8},
+                    {"int64", allow_int64},
+                    {"int128", allow_int128},
+                    {"float", allow_floating_points},
+                    {"date", allow_dates},
+                    {"date32", allow_date32},
+                    {"datetime", allow_datetimes},
+                    {"datetime64", allow_datetime64},
+                    {"string", allow_strings},
+                    {"decimal", allow_decimals},
+                    {"uuid", allow_uuid},
+                    {"enum", allow_enum},
+                    {"uuid", allow_uuid},
+                    {"dynamic", allow_dynamic},
+                    {"json", allow_JSON},
+                    {"nullable", allow_nullable},
+                    {"lcard", allow_low_cardinality},
+                    {"array", allow_array},
+                    {"map", allow_map},
+                    {"tuple", allow_tuple},
+                    {"variant", allow_variant},
+                    {"nested", allow_nested},
+                    {"ipv4", allow_ipv4},
+                    {"ipv6", allow_ipv6},
+                    {"geo", allow_geo}};
+
+             for (const auto & entry : split)
```

Comment:
We can remove `splitString` entirely and use `std::ranges::views::split(input, delimiter)` instead from C++20. That's more performant than extracting the strings and building an entire `std::vector` copying all `std::string`s from `input`